lib/active_model_serializers/adapter/json_api/resource_identifier.rb

```diff
@@ -2,11 +2,30 @@ module ActiveModelSerializers
   module Adapter
     class JsonApi
       class ResourceIdentifier
+        def self.type_for(class_name, serializer_type = nil, transform_options = {})
+          if serializer_type
+            raw_type = serializer_type
```

Comment:
I'm not sure why the current behavior of AMS is the transform the keys in the `serializer._type` but not pluralize them.  This just preserves that behavior.